== TEST (2019-03-25 22:06:24) 
-- CODE:
x = int(1998)
-- OUTPUT:
63.617
381.704
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-25 22:11:57) 
-- CODE:
n = int(input("1998: "))

soma = 0

while (n > 0):

    resto = n % 10
    n = n // 10
    soma = soma + resto


print("A soma dos números é: ", soma

-- ERROR:
File "XXXX", line 13
    
                                          ^
SyntaxError: unexpected EOF while parsing
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-25 22:12:51) 
-- CODE:
n = int(input("1998: "))

soma = 0

while (n > 0):

    resto = n % 10
    n = n // 10
    total= soma + resto


print(total)

-- OUTPUT:
9
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-25 22:14:03) 
-- CODE:
n = int(input("1998"))
soma = 0
while (n > 0)
resto = n % 10
n = n // 10
total= soma + resto
print(total)
-- ERROR:
File "XXXX", line 3
    while (n > 0)
                ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-25 22:14:13) 
-- CODE:
n = int(input("1998"))
soma = 0
while(n > 0)
resto = n % 10
n = n // 10
total= soma + resto
print(total)
-- ERROR:
File "XXXX", line 3
    while(n > 0)
               ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-25 22:15:39) 
-- CODE:
n = int(input("1998"))
soma = 0
while(i> 0)
resto = n % 10
n = n // 10
total= soma + resto
print(total)
-- ERROR:
File "XXXX", line 3
    while(i> 0)
              ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-25 22:16:02) 
-- CODE:

print(total)
-- EXECUTION TIME:
0.174567
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1602/842/main.py", line 2, in <module>
    print(total)
NameError: name 'total' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:20:17) 
-- CODE:
a = int(input("numero:"))

-- ERROR:
File "XXXX", line 3
    while(i> 0)
              ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:25:46) 
-- CODE:
a = int(input("numero:"))
n = int(input("Digite um número inteiro: "))

soma = 0

while (n > 0):

    resto = n % 10
    n = n // 10
    soma = soma + resto


print("A soma dos números é: ", soma)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    n = int(input())
EOFError: EOF when reading a line
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 00:25:51) 
-- CODE:
a = int(input("numero:"))
n = int(input("Digite um número inteiro: "))

soma = 0

while (n > 0):

    resto = n % 10
    n = n // 10
    soma = soma + resto


print("A soma dos números é: ", soma)
-- EXECUTION TIME:
0.133638
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1602/842/main.py", line 2, in <module>
    n = int(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:26:15) 
-- CODE:
a = int(input("numero:"))
n = int(input("Digite um número inteiro: "))

soma = 0

while (n > 0):

    resto = n % 10
    n = n // 10
    soma = soma + resto


print(soma)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    n = int(input())
EOFError: EOF when reading a line
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 00:26:19) 
-- CODE:
a = int(input("numero:"))
n = int(input("Digite um número inteiro: "))

soma = 0

while (n > 0):

    resto = n % 10
    n = n // 10
    soma = soma + resto


print(soma)
-- EXECUTION TIME:
0.15958
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1602/842/main.py", line 2, in <module>
    n = int(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:30:16) 
-- CODE:
n = int(input("Digite um número inteiro: "))
resto = n % 10
n = n // 10 
soma = soma + resto
print(soma)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    soma = soma + resto
NameError: name 'soma' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 00:30:25) 
-- CODE:
n = int(input("Digite um número inteiro: "))
resto = n % 10
n = n // 10 
soma = soma + resto
print(soma)
-- EXECUTION TIME:
0.150804
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1602/842/main.py", line 4, in <module>
    soma = soma + resto
NameError: name 'soma' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:33:47) 
-- CODE:
n = int(3141)
resto = n % 10
n = n // 10 
soma = soma + resto
print(soma)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    soma = soma + resto
NameError: name 'soma' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:34:12) 
-- CODE:
n = int(3141)
soma = 0
resto = n % 10
n = n // 10 
soma = soma + resto
print(soma)
-- OUTPUT:
1
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 00:34:21) 
-- CODE:
n = int(3141)
soma = 0
resto = n % 10
n = n // 10 
soma = soma + resto
print(soma)
-- EXECUTION TIME:
0.17777
-- TEST CASE 1:
---- input:
9173
---- correct output:
20
---- user output:
1
-- TEST CASE 2:
---- input:
1010
---- correct output:
2
---- user output:
1
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:34:51) 
-- CODE:
n = int(3141)
soma = 9
resto = n % 10
n = n // 10 
soma = soma + resto
print(soma)
-- OUTPUT:
10
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 00:34:55) 
-- CODE:
n = int(3141)
soma = 9
resto = n % 10
n = n // 10 
soma = soma + resto
print(soma)
-- EXECUTION TIME:
0.130185
-- TEST CASE 1:
---- input:
9173
---- correct output:
20
---- user output:
10
-- TEST CASE 2:
---- input:
1010
---- correct output:
2
---- user output:
10
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:35:38) 
-- CODE:
n = int(3141)
soma = 9
print(soma)
-- OUTPUT:
9
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 00:35:41) 
-- CODE:
n = int(3141)
soma = 9
print(soma)
-- EXECUTION TIME:
0.228151
-- TEST CASE 1:
---- input:
9173
---- correct output:
20
---- user output:
9
-- TEST CASE 2:
---- input:
1010
---- correct output:
2
---- user output:
9
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:36:14) 
-- CODE:
n = int(3141)
soma de n = 3+1+4+1
print(soma de n)
-- ERROR:
File "XXXX", line 2
    soma de n = 3+1+4+1
          ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 00:36:47) 
-- CODE:
n = int(3141)
soma de n = 3+1+4+1
print("soma de n")
-- ERROR:
File "XXXX", line 2
    soma de n = 3+1+4+1
          ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 09:59:48) 
-- CODE:
n = int(3141)

print(soma de n)
-- ERROR:
File "XXXX", line 3
    print(soma de n)
                ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 10:01:14) 
-- CODE:
n = int(3141)
%3//1%4//1
-- ERROR:
File "XXXX", line 2
    %3//1%4//1
    ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 10:03:04) 
-- CODE:
n = int(3141)
while(1)
-- ERROR:
File "XXXX", line 2
    while(1)
           ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 10:03:25) 
-- CODE:
n = int(3141)
while(1):
-- ERROR:
File "XXXX", line 3
    
             ^
SyntaxError: unexpected EOF while parsing
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-27 10:16:13) 
-- CODE:
num = int(input("numero:"))
u = num // 1 % 10
d = num // 10 % 10
c = num // 100 % 10
m = num // 1000 % 10
total = u + d + c + m
print(total)
-- OUTPUT:
20
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 10:16:36) 
-- CODE:
num = int(input("numero:"))
u = num // 1 % 10
d = num // 10 % 10
c = num // 100 % 10
m = num // 1000 % 10
total = u + d + c + m
print(total)
-- EXECUTION TIME:
0.179159
-- TEST CASE 1:
---- input:
9173
---- correct output:
20
---- user output:
20
-- TEST CASE 2:
---- input:
1010
---- correct output:
2
---- user output:
2
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 19:13:21) 
-- CODE:
num = int(input("numero:"))
u = num // 1 % 10
d = num //10 % 10
C = num // 100 % 10
m = num // 1000 % 10
total = u + d + c + m
print(total)
-- EXECUTION TIME:
0.115512
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1602/842/main.py", line 6, in <module>
    total = u + d + c + m
NameError: name 'c' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-27 19:13:40) 
-- CODE:
num = int(input("numero:"))
u = num // 1 % 10
d = num //10 % 10
c = num // 100 % 10
m = num // 1000 % 10
total = u + d + c + m
print(total)
-- EXECUTION TIME:
0.145642
-- TEST CASE 1:
---- input:
9173
---- correct output:
20
---- user output:
20
-- TEST CASE 2:
---- input:
1010
---- correct output:
2
---- user output:
2
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
